satpy/etc/composites/ahi.yaml

```diff
@@ -16,9 +16,50 @@ modifiers:
 
 composites:
   green:
+    # Deprecated green composite using deprecated class 'GreenCorrector'. Use 'hybrid_green' instead.
     compositor: !!python/name:satpy.composites.spectral.GreenCorrector
     # FUTURE: Set a wavelength...see what happens. Dependency finding
     #         probably wouldn't work.
+    prerequisites:
+      # should we be using the most corrected or least corrected inputs?
+      # what happens if something requests more modifiers on top of this?
```

Comment:
Did you get answers to these questions?

Reply:
Just to note that those questions have been in there for ages. Do't think we have an answer ;-)

Reply:
haha, ok, so rethorical questions really :)

Reply:
yeah, I don't know from when/whom this comes, so I just left it as it was for now.

Reply:
These comments are from @djhoese from [5 years ago](https://github.com/pytroll/satpy/commit/1ac2f79f277c1f2a1f4eb1791aa6831bf4a1528d#diff-b4a3d17fcd683bf50e4f051d8b7e9beaf234f3f513f5e36d20ebd556d9991bb0).

Reply:
no worries then

Reply:
I think these can be removed. These questions seem focused on how the dependency handling would behave. The other part of these questions is: is it correct to say that this is a plain "green" composite even though it is created from corrected/modified inputs? Would the result be worse if we used non-corrected inputs and then corrected the result? That might not work since pyspectral doesn't know about this imaginary/new green band.